instrumentation/jvm-metrics/src/test/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/JvmMetricsTest.java

```diff
@@ -64,6 +64,10 @@ void shouldRegisterOtelJvmMeters() {
     assertOtelMetricPresent("runtime.jvm.memory.used", "bytes");
     // thread metrics
     assertOtelMetricPresent("runtime.jvm.threads.peak", "threads");
+    // allocated memory metrics
+    assertOtelMetricPresent(AllocatedMemoryMetrics.METRIC_NAME, "bytes");
+    // Our custom GC metrics
+    assertOtelMetricPresent(GcMemoryMetrics.METRIC_NAME, "bytes");
```

Comment:
Is it worth trying to verify the type of these metrics?

Reply:
removed